### PR TITLE
Thu/refactor dataset creation parameters prod 13195

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by esasova — APPROVED** [review]
LGTM :+1:

**Review by nborde-CSM — APPROVED** [review]
One question but LGTM :+1:

File: src/services/config/ApiConstants.js [hunk 1]
```diff
@@ -59,56 +59,31 @@ export const DATASET_SOURCE_TYPE = {
...
-      en: 'Empty',
-      fr: 'Dataset vide',
-    },
+    labels: { en: 'Empty', fr: 'Dataset vide' },
     parameters: [],
   },
 ];
+
+DATASET_SOURCES.forEach((source) => {
+  source.tags = ['datasource'];
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply src/services/config/ApiConstants.js]: @nborde-CSM I think you commented an "outdated" version of the code, could you try to pull the branch and make sure you have the last version please ? :slightly_smiling_face:

[the remark this answers is not in the document]
Comment (csm-thu) [reply src/services/config/ApiConstants.js]: As discussed, this might be a change from an intermediary commit :slightly_smiling_face:

**Pull request merged**